### PR TITLE
New version: Treexy.DriverFusion version 11.0.0.0

### PR DESCRIPTION
**Pull request by TreexySoftware**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115121&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — CHANGES_REQUESTED** [review]
Looks like this installs 11.0.0.5

File: manifests/t/Treexy/DriverFusion/11.0.0.0/Treexy.DriverFusion.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Treexy.DriverFusion
+PackageVersion: 11.0.0.0
```
Comment (Trenly) [inline manifests/t/Treexy/DriverFusion/11.0.0.0/Treexy.DriverFusion.installer.yaml:5]:
```suggestion
PackageVersion: 11.0.0.5
```

File: manifests/t/Treexy/DriverFusion/11.0.0.0/Treexy.DriverFusion.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Treexy.DriverFusion
+PackageVersion: 11.0.0.0
```
Comment (Trenly) [inline manifests/t/Treexy/DriverFusion/11.0.0.0/Treexy.DriverFusion.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 11.0.0.5
```

File: manifests/t/Treexy/DriverFusion/11.0.0.0/Treexy.DriverFusion.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Treexy.DriverFusion
+PackageVersion: 11.0.0.0
```
Comment (Trenly) [inline manifests/t/Treexy/DriverFusion/11.0.0.0/Treexy.DriverFusion.yaml:5]:
```suggestion
PackageVersion: 11.0.0.5
```

**Review by Trenly — CHANGES_REQUESTED** [review]
After changing the PackageVersion, the folder will also need to be renamed

**Comment by TreexySoftware** [comment]
> After changing the PackageVersion, the folder will also need to be renamed

We don't use the last digit, we market as 11.x.

**Comment by TreexySoftware** [comment]
> Looks like this installs 11.0.0.5


After changing the PackageVersion, the folder will also need to be renamed

Yes, but we don't advertise the last digit, we market as 11.x.

**Comment by TreexySoftware** [comment]
> Looks like this installs 11.0.0.5

Yes, but we don't advertise the last digit, we market as 11.x.

**Comment by Trenly** [comment]
> > Looks like this installs 11.0.0.5
> 
> Yes, but we don't advertise the last digit, we market as 11.x.

Then you need to add `AppsAndFeaturesEntries` with the `DisplayVersion`, since winget uses the value written to control panel to determine what version is installed and if any updates are available

**Pull request closed without merging**